### PR TITLE
Updates camera config

### PR DESCRIPTION
**Pull request by tayloryoung6396**

### CODE REVIEW

File: module/input/Camera/data/config/igus2/Cameras/Left.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 format:
   width: 1280
   height: 1024
-  pixel: BayerBG8
+  pixel: BayerRG8
 
 lens:
-  radiansPerPixel: 0.00245
+  radiansPerPixel: 0.0024242              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 4.8um / 1.98mm = 2.4242e-3
```
Comment (TrentHouliston) [inline module/input/Camera/data/config/igus2/Cameras/Left.yaml]: Do it properly!

Comment (TrentHouliston) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: https://medium.com/@kennethjiang/calibrate-fisheye-lens-using-opencv-333b05afa0b0

Comment (tayloryoung6396) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: No

Comment (TrentHouliston) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: Seems you have committed other things then!

Comment (tayloryoung6396) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: THAT'S ONLY SO THEY'RE ALL THE SAME :P

File: module/input/Camera/data/config/igus2/Cameras/Right.yaml [hunk 2]
```diff
@@ -9,12 +9,12 @@ format:
   pixel: BayerRG8
 
 lens:
-  radiansPerPixel: 0.0024242              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 4.8um / 1.98mm = 2.4242e-3
+  radiansPerPixel: 0.00245
```
Comment (Bidski) [inline module/input/Camera/data/config/igus2/Cameras/Right.yaml]: Preserve this comment. This tells you how to calculate this value

File: module/input/Camera/data/config/igus2/Cameras/Left.yaml (hunk 1 above)
Comment (Bidski) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: Calibration should be a separate PR

Comment (tayloryoung6396) [reply module/input/Camera/data/config/igus2/Cameras/Left.yaml]: Happy now!

File: module/input/Camera/data/config/igus1/Cameras/Left.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ format:
   pixel: BayerRG8
 
 lens:
-  radiansPerPixel: 0.0026768              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 5.3um / 1.98mm = 2.6768e-3
+  radiansPerPixel: 0.0024242              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 5.3um / 1.98mm = 2.6768e-3
```
Comment (Bidski) [inline module/input/Camera/data/config/igus1/Cameras/Left.yaml]: Correct the coments!

**Pull request merged**